pkg/k8s/api_test.go

```diff
@@ -132,7 +132,7 @@ status:
 		},
 		{
 			desc:     "Pod has a running container",
-			expected: "Running",
+			expected: "NotReady",
```

Comment:
Nice, it seems this is more coherent given the the phase is still "Pending"

Reply:
Actually no, I was looking at the pod spec above this... is this pod's status missing something that makes it NotReady?